src/compose/service.coffee

```diff
@@ -0,0 +1,343 @@
+_ = require 'lodash'
+path = require 'path'
+{ checkTruthy, checkInt } = require '../lib/validation'
+updateLock = require '../lib/update-lock'
+constants = require '../lib/constants'
+conversions =  require '../lib/conversions'
+
+validRestartPolicies = [ 'no', 'always', 'on-failure', 'unless-stopped' ]
```

Comment:
"no" isn't a valid policy

Reply:
Oh, you're right! I thought that it was because it's a valid option in docker compose, and it's also what docker returns when you inspect a container without restart policy. I changed `toContainerConfig` to not set any restart policy in this case - but it's still a valid policy when it comes as an input to the Service constructor.